ospd_openvas/daemon.py

```diff
@@ -1116,6 +1116,38 @@ def report_openvas_timestamp_scan_host(
             )
             return
 
+    def openvas_process_is_alive(
```

Comment:
The name of the method should be
```suggestion
    def is_openvas_process_alive(
```